src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -518,6 +521,18 @@ protected VariantContext removeAnnotations(VariantContext vc) {
                                                                       GATKVCFConstants.FISHER_STRAND_KEY));
 
         builder.rmAttributes(rmAnnotationList);
+        return builder.make();
+    }
+
+    protected VariantContext addChromosomeCountsAnnotations(VariantContext vc) {
+
+        final VariantContextBuilder builder = new VariantContextBuilder(vc);
+
+        // add AC/AF/AD annotations
+        Set<String> founderIds = new HashSet<>();
+        Map<String, Object> attributes = new HashMap<>(vc.getAttributes());
+        Map<String, Object> chromosomeCounts = calculateChromosomeCounts(vc, attributes, true, founderIds);
+        builder.putAttributes(chromosomeCounts);
```

Comment:
the docs on this are confusing (about removing stale attributes, why we pass things in and out, etc) so I'm not sure we need to call putAttribues here...

but the bigger question would be, why not just use the GATK annotation infrastructure?